src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
```

Comment:
you've mentioned two times bucket in your previous sentence, so "This bucket" is which bucket? better to use "which is ..."